CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [ryu](https://github.com/dtolnay/ryu) is used to serialize `f32` and `f64`
 - Added missing implementation for `serialize_bytes` method
 
+### Breaking
+- Heapless dependency updated to 0.6.1 - publicly exporting heapless for external use.
+
```

Comment:
```suggestion
- Publicly re-export `heapless` dependency to allow for external usage.
- [breaking-change] Heapless dependency updated to 0.6.1
```
Hmm, "Breaking" is not a separate category as per keepachangelog.com.
How about something like this?